core/urls.py

```diff
@@ -18,13 +18,15 @@
 from rest_framework.documentation import include_docs_urls
 
 from core.authors.friend_request_view import handle_follow_request as friend_request
+from core.posts.posts_view import handle_posts as handle_posts
 from core.authors.login_view import login
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('users/', include('core.users.urls')),
     path('author/', include('core.authors.urls')),
     path('posts/', include('core.posts.urls')),
+    path('handleposts/', handle_posts, name='handleposts'),
```

Comment:
I think the [path to make a new post](https://github.com/abramhindle/CMPUT404-project-socialdistribution/blob/master/project.org#requirements) is /posts/POST_ID, but [this](https://github.com/abramhindle/CMPUT404-project-socialdistribution/blob/master/example-article.json#L259) says that path is meant for querying for a specific post. You might want to ask for clarification on that on the forums

Regardless, we still need a PUT method on /posts/POST_ID